src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let total = days * 40;
```

Comment:
it will be more informative to use the "totalRent" as name of the variable "total"